app/services/cognito/sign_in_user.rb

```diff
@@ -77,5 +79,9 @@ def login_user_cognito(client_id, client_secret)
     def cookies_should_be_enabled
       errors.add(:base, :cookies_disabled) if @cookies_disabled
     end
+
+    def redirect_uri_known
+      errors.add(:base, :redirect_uri_mismatch) unless ENV['CALLBACK_URLS'].include?(@redirect_uri)
```

Comment:
What are the callback URLs?

Reply:
The supplier Callback URL set up in Cognito. On Tail Spend we have 2, that is why this is an array

Reply:
What ITHC issue is this fixing out of curiosity?

Reply:
https://crowncommercialservice.atlassian.net/browse/TS-133

For some reason this didn't come up in the PMP ITHC

Reply:
Thanks Iullia, I will do these updates in PMP too